frontend/src/pages/Faculty/student/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react'
+import StudentList from './studentList'
+import ProfileOpen from './profileOpen'
+
+export default class Student extends Component {
+    state = {
+        students: null,
+        selectedStudentData: null,
+        page: 1,
+        countData: 0,
+        studentNo: 0,
+        searchQuery: ''
+    };
+    componentDidMount() {
+        this.getStudentData();
+    }
+
+
+
+
+    getStudentData = () => {

```

Comment:
api queries should be in their own folder (logistics/services i believe).
Also the other queries are written differently (they use the fetch function)